docs/reference/api/pages.md

```diff
@@ -371,6 +372,34 @@ The successful PATCH request returns the complete patched document. See the [pag
 
 The trash is part of the overall page tree in order to maintain the nesting structure. As such, there is not only a simple `trash` property to set `true`. Instead, set `_targetId` to `_trash` and `_position` to `lastChild` (or another position within the trash). You may similarly move pages out of the trash by moving them to a position relative to another page that is not in the trash.
 
+## `GET /:_url?apos-refresh=1`
+
+Including the `apos-refresh=1` query parameter value on an Apostrophe page URL returns the rendered HTML from the `refreshLayout.html` template, which excludes the wrapping markup from the `outerLayoutBase.html` template file outside of the `[data-apos-refreshable]` element. Apostrophe UI uses this parameter to refresh content during editing.
+
+**We do not recommend making edits to `refreshLayout.html` for the sake of API requests as this template is critical to normal content editing.**
```

Comment:
I want to note here that I'm intentionally not giving additional direction regarding alternatives. We want to keep this as relatively pure reference material. A separate guide could offer additional suggestions.